versions.tf

```
@@ -1,3 +1,3 @@
 terraform {
-  required_version = ">= 0.12.14"
+  required_version = ">= 0.12.19"
```

```suggestion
  required_version = "~> 0.12.19"
```

Should really avoid accidentally using `0.13` until we've validated on it